spec/system/fast_edit_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+describe "Fast edit", type: :system, js: true do
+  let(:topic_page) { PageObjects::Pages::Topic.new }
+  let(:fast_editor) { PageObjects::Components::FastEditor.new }
+  fab!(:topic) { Fabricate(:topic) }
+  fab!(:post) { Fabricate(:post, topic: topic) }
+  fab!(:post_2) { Fabricate(:post, topic: topic, raw: "It 'twas a great time!") }
+  fab!(:current_user) { Fabricate(:admin) }
+
+  before do
+    current_user.activate
```

Comment:
I don't think we strictly _have_ to do this, try removing it, if it fails then we know we do need it :)